app/services/idv/vendor_result.rb

```diff
@@ -22,5 +22,15 @@ def initialize(success:, errors:, reasons:, session_id:, normalized_applicant:)
     def success?
       success
     end
+
+    # rubocop:disable Metrics/AbcSize
+    def ==(other)
+      success == other.success &&
+        errors == other.errors &&
+        reasons == other.reasons &&
+        session_id == other.session_id &&
+        normalized_applicant.as_json == other.normalized_applicant.as_json
+    end
+    # rubocop:enable Metrics/AbcSize
```

Comment:
I put the comment on the wrong line. I meant what does the method do? Sounds like it's mainly to facilitate testing? Is this method used anywhere besides tests?